libretroshare/src/util/cxx11retrocompat.h

```diff
@@ -19,15 +19,15 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.       *
  *                                                                             *
  *******************************************************************************/
-#pragma once
+//#pragma once
```

Comment:
commenting this whole bloc out seems dangerous to me given that is has been put here to allow to compile on other systems. At least put a proper #ifndef MACOS around.

Reply:
`#pragma once` should be supported by any decent compiler out there